src/server_manager/web_app/app.ts

```diff
@@ -1248,4 +1255,13 @@ export class App {
     document.documentElement.setAttribute('dir', languageDir);
     window.localStorage.setItem('overrideLanguage', languageCode);
   }
+
+  private createLocationModel(cityId: string, regionIds: string[]): Location {
+    return {
+      id: regionIds[0],
```

Comment:
If regionIds.length == 0 this won't throw an error, I think, but perhaps we can be more explicit with the ternary operator:  `id: regionIds.length > 0 ? regionIds[0] : null`

Reply:
Sounds good